tests/floris_interface_test.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+
+from floris.tools.floris_interface import FlorisInterface
+
+
+TEST_DATA = Path(__file__).resolve().parent / "data"
+YAML_INPUT = TEST_DATA / "input_full_v3.yaml"
+JSON_INPUT = TEST_DATA / "input_full_v3.json"
+
+
+def test_read_json():
+    fi = FlorisInterface(configuration=JSON_INPUT)
+    assert isinstance(fi, FlorisInterface)
+
+
+def test_read_yaml():
+    fi = FlorisInterface(configuration=YAML_INPUT)
+    assert isinstance(fi, FlorisInterface)
+
+
+def test_calculate_wake():
+    pass
```

Comment:
The easiest test to pass.

Reply:
These are just brutal reminders for ourselves ;)